DcShare/Controller/ShareViewController.swift

```diff
@@ -65,11 +66,16 @@ class ShareViewController: SLComposeServiceViewController {
         if dbHelper.currentDatabaseLocation == dbHelper.sharedDbFile {
             dcContext.logger = self.logger
             dcContext.openDatabase(dbFile: dbHelper.sharedDbFile)
-            selectedChatId = dcContext.getChatIdByContactId(contactId: Int(DC_CONTACT_ID_SELF))
-            if let chatId = selectedChatId {
-                selectedChat = dcContext.getChat(chatId: chatId)
+            isAccountConfigured = dcContext.isConfigured()
+            if isAccountConfigured {
+                isAccountConfigured = true
+                selectedChatId = dcContext.getChatIdByContactId(contactId: Int(DC_CONTACT_ID_SELF))
+                if let chatId = selectedChatId {
+                    selectedChat = dcContext.getChat(chatId: chatId)
+                }
             }
             reloadConfigurationItems()
+            validateContent()
```

Comment:
just to get an idea about the flow: why was the call to `validateContent()` not needed before?

Reply:
because validateContent gets called earlier in the life cycle when we don't yet check if the account was configured or not. 
Before this PR changes we didn't check the account configuration at all. 
We don't move the configuration check to an earlier slot in the life cycle, because 'heavier' processings should not happen before `presentationAnimationDidFinish()`

Reply:
thanks for explanations!